ifrs17-template/Test/ScenarioDataImportTest.ipynb

```diff
@@ -0,0 +1,868 @@
+            "source": [
+                "diffs.Where(x => actualEstimateTypes.Contains(x.EstimateType)).ToArray().Length.Should().NotBe(0);",
+                "\ndiffs.Where(x => x.EstimateType == EstimateTypes.CU).ToArray().Length.Should().Be(0);",
+                "\ndiffs.Where(x => x.EstimateType == EstimateTypes.DA && Math.Abs(x.Value) > Precision).ToArray().Length.Should().Be(0);"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Update of Deferrable Actuals with a Scenario Coverage Units Import"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "Update CU cashflow for scenario and check that the Amortization Factor (and hence the Deferrable Actual) is updated"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var cashflowsScenarioUpdateCU = @\"",
+                "\n@@Main",
+                "\nReportingNode,Year,Month,Scenario",
+                "\nCH,2020,12,Test",
+                "\n@@Cashflow",
+                "\nDataNode,AmountType,AocType,Novelty,AccidentYear,Values0,Values1,Values2,Values3,Values4,Values5,Values6,Values7,Values8,Values9,Values10,Values11,Values12,Values13,Values14,Values15,Values16,Values17,Values18,Values19,Values20,Values21,Values22,Values23",
+                "\nDT1.1,CU,CL,C,,-17,-17,-17,-17,-17,-17,-17,-17,-17,-17,-17,-17,-17,-6.5,-6.5,-6.5,-6.5,-6.5,-6.5,-6.5,-6.5,-6.5,-7.5,-8.5\";"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await Import.FromString(cashflowsScenarioUpdateCU).WithFormat(ImportFormats.Cashflow).WithTarget(DataSource).ExecuteAsync()"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await DataSource.Partition.SetAsync<PartitionByReportingNodeAndPeriod>(argsScenario);",
+                "\nvar ivsScenarioCashflowUpdateCU = await DataSource.Query<IfrsVariable>().ToArrayAsync();",
+                "\nvar diffs = ivsScenarioCashflowUpdateCU.Except(ivsScenarioInitActuals, IfrsVariableComparer.Instance());"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "diffs.Where(x => x.EstimateType == EstimateTypes.DA).ToArray().Length.Should().NotBe(0);",
```

Comment:
I would add : 
diffs.Where(x => x.EstimateType == EstimateTypes.C || x.EstimateType == EstimateTypes.L).ToArray().Length.Should().NotBe(0);

Reply:
done